src/modules/rep.ts

```diff
@@ -158,6 +158,12 @@ export class RepModule extends Module {
 		description: 'See who has the most reputation',
 	})
 	async leaderboard(msg: Message) {
+		function getLeaderboardEmoji(index: number) {
+			if (index === 0) return ':first_place:';
+			else if (index === 1) return ':second_place:';
+			else if (index === 2) return ':third_place:';
+			else return ':white_small_square:';
```

Comment:
I think it would be cleaner if it was just an empty line here then `return ':small_white_square:'` (removing the `else`)

Reply:
Alternatively...
```ts
function getLeaderboardEmoji(index: number) {
  return [':first_place:', ':second_place:', ':third_place:'][index] || ':small_white_square:';
}